src/test/resources/features/Login.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Login feature
+
+  Scenario: Login Success
+    Given I open browser
+    When I enter email "email"
+    And I enter password "password"
+    And I submit
+    Then I am logged in
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/218272858-4a821f24-8eb6-42c6-8a7b-2600305f6dc3.png)